go.mod

```diff
@@ -1,6 +1,8 @@
 module sigs.k8s.io/cluster-api-provider-openstack
 
-go 1.22.0
+go 1.23.0
+
+toolchain go1.23.8
```

Comment:
I don't think we want that in a release version.
Let's let the bot doing this update on Mondays/Thursdays if possible?

Reply:
I agree that we shouldn't change this.  The reason for the initial change was due to the CVE that was blowing up our scanning :)

https://avd.aquasec.com/nvd/2025/cve-2025-22870/